android/app/src/googleplay/java/org/fossasia/openevent/utils/ClusterRenderer.java

```diff
@@ -0,0 +1,38 @@
+package org.fossasia.openevent.utils;
+
+import android.content.Context;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.Marker;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.google.maps.android.clustering.ClusterManager;
+import com.google.maps.android.clustering.view.DefaultClusterRenderer;
+
+import org.fossasia.openevent.R;
+
+/**
+ * Created by rohanagarwal94 on 4/7/17.
+ */
+public class ClusterRenderer extends DefaultClusterRenderer<MicrolocationClusterWrapper> {
```

Comment:
Instead of creating a concrete class, this could have been accomplished using anonymous inner class. If the class is not to be reused, it's better to use anon classes